sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakePathAsyncClient.java

```diff
@@ -413,6 +416,7 @@ public DataLakeServiceVersion getServiceVersion() {
     public Mono<Response<PathProperties>> getPropertiesWithResponse(DataLakeRequestConditions requestConditions) {
         try {
             return blockBlobAsyncClient.getPropertiesWithResponse(Transforms.toBlobRequestConditions(requestConditions))
+                .onErrorMap(ex -> DataLakeImplUtils.transformBlobStorageException((BlobStorageException) ex))
```

Comment:
Should this be after the `map`?

Reply:
For some reason putting onErrorMap after gives me a build issue telling me that it expects the method to return a Mono<SimpleResponse<PathProperties>> so any method that maps to a simple response I just do later